### DISCUSSION TITLE
OpenCEX for Hong Kong VASP license

### DISCUSSION THREAD

**Discussion by joequant** [Polygant/OpenCEX#1] [General]
Hi all,

User joequant here.  Working on getting my OpenCEX up and running.  The reason for this is that Hong Kong is introducing crypto licensing but any company doing a crypto exchange before 1 June 2023 will be eligible to apply for a transitional license. 

See

https://github.com/joequant/bitquant-vasp

**Comment by Stan-Polygant** [comment]
Hi,

Great that you are using OpenCEX for building your exchange! 
We wish you luck, please share the updates how's it going.
